src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -263,9 +263,11 @@ public function configureOptions(OptionsResolver $resolver)
                 return $choices;
             }
 
-            ChoiceType::normalizeLegacyChoices($choices, $choiceLabels);
+            if (null === $choices) {
+                $choices = array();
```

Comment:
@stof here is `null` handling. But should we deprecate accepting the `null` value? Otherwise, 3.0 misses a "choices" normalizer also, isn't it?
https://github.com/symfony/symfony/pull/16796/files#diff-11ee91a32c601e8e1d509117556d53b3R400